src/types.ts

```diff
@@ -59,4 +59,6 @@ export type Assets = {
         [key: string]: string[]
     },
     clientsOrig: Asset[],
-}
\ No newline at end of file
+}
+
+export type AssetTypes = 'rules' | 'rulesConfigs' | 'hooks' | 'pages' | 'databases' | 'clientGrants' | 'resourceServers' | 'clients' | 'connections' | 'tenant' | 'emailProvider' | 'emailTemplates' | 'guardianFactors' | 'guardianFactorProviders' | 'guardianFactorTemplates' | 'migrations' | 'guardianPhoneFactorMessageTypes' | 'guardianPhoneFactorSelectedProvider' | 'guardianPolicies' | 'roles' | 'actions' | 'organizations' | 'triggers' | 'attackProtection' | 'branding'
```

Comment:
The primary addition of this PR.

Reply:
@sergiughf please excuse the lack of a new line EOF, I'm making a linter change #461 🙈